app/renderer/js/main.ts

```diff
@@ -370,9 +370,9 @@ class ServerManagerView {
 					}
 					this.showLoading(this.loading[this.tabs[this.activeTabIndex].webview.props.url]);
 				},
-				onNetworkError: this.openNetworkTroubleshooting.bind(this),
+				onNetworkError: (index: number) => this.openNetworkTroubleshooting(index),
 				onTitleChange: this.updateBadge.bind(this),
-				nodeIntegration: false,
+				nodeIntegration: true,
```

Comment:
I believe that is enables node integration for servers added, if so, I think we should avoid this because it gives access to node api to a url we are loading which isn't our code.

I think this is required for network.js and the workaround for that would be to require it in preload.js; that will allow us to not enable `nodeIntegration`

Reply:
@kanishk98 ^^

Reply:
Sure, I'll just make the change. Thanks for pointing this out. :)